pkgs/by-name/go/godot_4/package.nix

```diff
@@ -63,14 +63,21 @@ let
 
   attrs = finalAttrs: rec {
     pname = "godot4${suffix}";
-    version = "4.3-stable";
-    commitHash = "77dcf97d82cbfe4e4615475fa52ca03da645dbd8";
+    version = "4.4-stable";
 
     src = fetchFromGitHub {
       owner = "godotengine";
       repo = "godot";
-      rev = commitHash;
-      hash = "sha256-v2lBD3GEL8CoIwBl3UoLam0dJxkLGX0oneH6DiWkEsM=";
+      rev = version;
```

Comment:
```suggestion
      tag = version;
```

Reply:
Done.  I guess the difference here is that `tag` will match the tag explicitly `refs/tags/${tag}` rather than possibly being ambiguous?